pubsub/azurepubsub/azurepubsub.go

```diff
@@ -289,7 +289,14 @@ type subscription struct {
 
 // SubscriptionOptions will contain configuration for subscriptions.
 type SubscriptionOptions struct {
-	ListenerTimeout time.Duration
+	ListenerTimeout            time.Duration
```

Comment:
Please add a docstring for `ListenerTimeout`, including what the default is. IIUC, this is how long it will wait to try to get a full batch of messages. 1 second as the default seems a bit long to me; e.g., if `maxMessages` is 10, but only a single message arrives, we'll wait for a full second before returning it?

BTW, other drivers have something similar but haven't exposed it publicly; e.g. `rabbitpubsub` tries to get to `maxMessages` for 50ms:
https://github.com/google/go-cloud/blob/master/pubsub/rabbitpubsub/rabbit.go#L566

So you could also consider removing this option and making it a constant. We can always expose it later if needed.

Reply:
Yes, let's hardcode this for now.

Reply:
Option removed, btw the point of having it public is to allow users to configure if the default was non-optimal. An acceptable timeout **will** differ from environment to environment due to many factors. In any event, the setting is removed.